File: Base/Desktop/Allors/Client/Base/Workspace/Data/Response.ts
namespace Allors.Data {
    export interface Response {
        userSecurityHash: string;

        objects?: string[][];

        namedObjects?: { [id: string]: string; };
        namedCollections?: { [id: string]: string[]; };
        namedValues?: { [id: string]: any; };
    }
}

File: Base/Desktop/Allors/Server/Base/Workspace/Response.cs
namespace Allors.Web
{
    using System.Collections.Generic;

    public class Response
    {
        public string UserSecurityHash { get; set; }

        public string[][] Objects { get; set; }

        public Dictionary<string, string> NamedObjects { get; set; }

        public Dictionary<string, string[]> NamedCollections { get; set; }

        public Dictionary<string, object> NamedValues { get; set; }
    }
}